ui/projects/setup/src/splunk/splunk.service.ts

```diff
@@ -0,0 +1,87 @@
+import { Injectable } from "@angular/core";
+import {
+  ConfigurationFileStanzaEntity,
+  Index,
+  SplunkJsServiceBuilder,
+} from "./splunk-js.service";
+import { AppConfig } from "../config";
+
+@Injectable({
+  providedIn: "root",
+})
+export class SplunkService {
+  private readonly globalNamespaceService = new SplunkJsServiceBuilder({
+    owner: "-",
+    app: "-",
+    sharing: "app",
+  });
+  private readonly bitwardenAppService;
+
+  constructor(config: AppConfig) {
+    this.bitwardenAppService = new SplunkJsServiceBuilder({
+      owner: "nobody",
+      app: config.appName,
+      sharing: "app",
+    });
+  }
```

Comment:
Does the namespace distinguish between global Splunk configuration (e.g. indexes) and Bitwarden app configuration (e.g. client ID and secret)?